site/src/docs/components/login/usage.mdx

```diff
@@ -556,6 +620,41 @@ All `Session poller` settings, properties, methods and signals are detailed on t
 
 Detailed information about these hooks is listed on the <ApiPageAnchorLink anchor="session-poller-hooks">API page</ApiPageAnchorLink>.
 
+#### GraphQL module
+
+Loading data from a graphQL server may require user authentication and api tokens. The GraphQL module can be set to listen to the Api token client and start querying when api tokens are ready. It can pick the correct api token and set it to the query headers.
+
+The module can also be used without api tokens and can be controlled manually.
+
+##### Requirements
+
+To execute a query, the following requirements must be met:
+
+- A graphQLClient (ApolloClient) must be configured and passed to the `GraphQL module`.
+- A query document must be provided.
+
+Detailed information about authenticated queries is on the <ApiPageAnchorLink anchor="requirements-2">API page</ApiPageAnchorLink>.
```

Comment:
here written API. Consistency :)

Reply:
This is consistent with all other "API page" link texts. 😄  There are 24 of them.